### PR TITLE
Initial edits based on NIAID feedback

### PR DESCRIPTION
**Pull request by tom6931**
I've added some new issues to trouble shooting that should answer Philip's questions and made some edits to the quickstart.

### CODE REVIEW

File: docs/source/quickstart.rst [hunk 1]
```diff
@@ -23,11 +23,14 @@ The minimal configuration requires:
 
 #. Google Cloud Platform (GCP) parameters (:ref:`project <elb_gcp_project>`, :ref:`region <elb_gcp_region>`, and :ref:`zone <elb_gcp_zone>`),
 
-#. :ref:`query sequences <elb_queries>` in a single file or tarball, a :ref:`GCP bucket for results <elb_results_bucket>`,
+#. :ref:`query sequences <elb_queries>` in a single file or tarball, 
+
+#. a :ref:`GCP bucket for results <elb_results_bucket>`,  The name of this bucket will start with gs://
```
Comment (christiam) [inline docs/source/quickstart.rst]:
```suggestion
#. a :ref:`GCP bucket for results <elb_results_bucket>`;  the name of this bucket must start with gs://
```

File: docs/source/troubleshooting.rst [hunk 2]
```diff
@@ -42,3 +42,45 @@ resources created by ElasticBLAST. Your input file(s) will not be modified.
...
+Run
+
+.. code-block:: bash
+
+    python --version 
+
+to see which version of python you have (or if it's even installed).  If python is not found or
+it's not one of the supported versions (3.6, 3.7 or 3.8), you'll need to install it (3.8 recommended). 
+If one of versions from 3.6, 3.7 or 3.8 is found, then you can try the corresponding elastic-blast 
+script (e.g., elastic-blast3.8).
```
Comment (christiam) [inline docs/source/troubleshooting.rst]:
```suggestion
script (e.g., `elastic-blast3.8`).
```

File: docs/source/troubleshooting.rst [hunk 3]
```diff
@@ -42,3 +42,45 @@ resources created by ElasticBLAST. Your input file(s) will not be modified.
...
+
+
+I get a message about not being able write into the bucket with submit command
+------------------------------------------------------------------------------
+
+Things to check:
+
+* The project name is correct.  The project name and not the project ID is required in the config file.  It should correspond to what is returned by the command:
```
Comment (christiam) [inline docs/source/troubleshooting.rst]:
```suggestion
* The project ID is correct.  The project ID and not the project name or project number is required in the config file.  It should correspond to what is returned by the command:
```

**Review by christiam — COMMENTED** [review]
Thanks, the changes overall look good to me, though I made a couple of suggestions.

File: docs/source/troubleshooting.rst (hunk 3 above)
Comment (boratyng) [reply docs/source/troubleshooting.rst]: What needs to be provided in elastic-blast ini file is project ID, not name. GCP projects have a name, id, and number. Project name is a human readable title, project id is a user assigned id that can be used by APIs, and the number is a number that Google assigns to the project: https://cloud.google.com/resource-manager/docs/creating-managing-projects. Many people assume that id is a number, but it is not how Google defines these.

File: docs/source/quickstart.rst [hunk 4]
```diff
@@ -100,11 +108,20 @@ Run the command below to download the results
...
 
 
+The delete command will take a few minutes to run as it needs to shut the cluster down.
+You may verify that your cluster has been deleted by running: 
+
+.. code-block:: bash
+
+  gcloud container clusters list 
+
+This will show all clusters running in your project (even from other users).  If nothing is returned, then no clusters are running.
```
Comment (boratyng) [inline docs/source/quickstart.rst:127]: Should we also talk about persistent disk here and provide command lines to manually delete both?

File: docs/source/quickstart.rst [hunk 5]
```diff
@@ -100,11 +108,20 @@ Run the command below to download the results
...
 
     ./elastic-blast delete --cfg ${CONFIG_FILE} --loglevel DEBUG
 
 
+The delete command will take a few minutes to run as it needs to shut the cluster down.
+You may verify that your cluster has been deleted by running: 
+
+.. code-block:: bash
+
+  gcloud container clusters list
```
Comment (boratyng) [inline docs/source/quickstart.rst]:
```suggestion
  gcloud container clusters list --project <your-gcp-project-id>
```
The default project may not be set in user's gcloud config. Then the `--project` option must be provided.

File: docs/source/troubleshooting.rst (hunk 3 above)
Comment (boratyng) [inline docs/source/troubleshooting.rst]: Default project name/id has nothing do with whether one can write to a bucket. GCP user must have permissions to read/write to the bucket.

File: docs/source/troubleshooting.rst [hunk 6]
```diff
@@ -42,3 +42,45 @@ resources created by ElasticBLAST. Your input file(s) will not be modified.
...
+
+.. code-block:: bash
+
+   gcloud config get-value project
```
Comment (boratyng) [inline docs/source/troubleshooting.rst]: This code block should be removed. Default GCP project does not have to be set for elastic-blast to run and users may not have it set. Also, it has nothing to with ability to read/write to buckets.

**Review by boratyng — CHANGES_REQUESTED** [review]

File: docs/source/troubleshooting.rst (hunk 3 above)
Comment (christiam) [reply docs/source/troubleshooting.rst]: @boratyng is correct, I updated my suggestion accordingly.

File: docs/source/quickstart.rst (hunk 5 above)
Comment (tom6931) [reply docs/source/quickstart.rst]: Done

File: docs/source/troubleshooting.rst (hunk 6 above)
Comment (tom6931) [reply docs/source/troubleshooting.rst]: Removed

File: docs/source/troubleshooting.rst (hunk 3 above)
Comment (tom6931) [reply docs/source/troubleshooting.rst]: Removed (no longer relevant).

**Comment by tom6931** [comment]
I think Greg's  changes are resolved, but I still cannot merge.  Do I need to wait for Greg to verify or did I miss something? thanks!

File: docs/source/quickstart.rst (hunk 5 above)
Comment (boratyng) [reply docs/source/quickstart.rst]: I do not think these changes were applied.

Comment (tom6931) [reply docs/source/quickstart.rst]: You are right.  I had it locally on my mac (so saw it on my preview) but had failed to commit it.  I *think* this is done.  thanks.

**Review by boratyng — APPROVED** [review]

File: docs/source/quickstart.rst (hunk 4 above)
Comment (tom6931) [reply docs/source/quickstart.rst:127]: This is fine, but I'm going to start a new round of changes on Wednesday (aiming to be done by 8/14), and I'd prefer to leave it until then.

**Pull request merged**